Chat/ClientApp/src/components/MainArea.tsx

```diff
@@ -3,10 +3,17 @@ import { Image } from '@fluentui/react';
 import defaultImg from '../assets/default.png';
 import { staticImageStyle, staticAreaStyle} from './styles/ChatScreen.styles';
 
-export default (props: any): JSX.Element => {
+export interface MainScreenProps {
+  updateContent(content: string): void;
+}
+
+export default (props: MainScreenProps): JSX.Element => {
   const imageProps = { src: defaultImg.toString() };
-  return (
-    <div className={staticAreaStyle}>
+  const { updateContent } = props;
+
+  const getComponent = () => {
```

Comment:
From JS perspective, this change is equivalent to the previous code state.

Reply:
Yep I agree. As per comment above I will introduce the ability to change what displays on here